packages/sdk/src/index.ts

```diff
@@ -93,42 +94,48 @@ export class Gomu {
   }
 
   async makeOrder(params: MakeOrderParams): Promise<Order[]> {
-    // @ts-ignore
     return Promise.all(
       Object.entries(this.marketplaces)
-        .filter(([_, marketplace]) => marketplace)
-        .map(async ([marketplaceName, marketplace]) => ({
-          marketplaceName,
-          marketplaceOrder: await marketplace.makeOrder(params),
-        }))
+        .filter(([marketplaceName, marketplace]) => {
+          if (
+            params.marketplaces?.length &&
+            !params.marketplaces.includes(marketplaceName as MarketplaceName)
```

Comment:
I don't know why, but using includes on typed array value forces type matching for value that will be used, which doesn't makes a lot of sense, because why it would care, lol?

Reply:
Because typescript somehow doesn't recognise that `marketplaceName` will only be `"opensea"` or `"trader"` That's why I have quite a number of ts-ignore around. 😞 

Or if you want it more succinct.
```typescript
.filter(([marketplaceName, marketplace]) => {
          return (
            !marketplaces?.length ||
            (marketplaces.includes(marketplaceName as MarketplaceName) &&
              marketplace)
          );
```

Reply:
This one breaks my brain, I'll stick with mine for now if it's ok.